docs/local-development.md

```diff
@@ -0,0 +1,87 @@
+1. Install Ruby. Choose the version [in the `.ruby-version` file](../.ruby-version)
+1. Skip to the [set up local environment section](#set-up-local-environment). Your other dependencies will be installed in that step.
+
+If not using macOS:
+
+1. To start, make sure you have the following dependencies installed and a working development environment:
+
+    - [rbenv](https://github.com/rbenv/rbenv) (lets you install and switch between different versions of Ruby)
+    - Ruby. Choose the version [in the `.ruby-version` file](../.ruby-version)
+    - [PostgreSQL](http://www.postgresql.org/download/)
+    - [PostGIS](https://postgis.net/documentation/getting_started/#installing-postgis)
+        - Note: if you run into errors installing `postgis` or creating the database, check [the troubleshooting docs](./troubleshooting.md#errors-related-to-the-databse).
+    - [Redis 7+](http://redis.io/)
+
+2. You will need to install openssl version 1.1:
+
+    - Run `brew install openssl@1.1`
+
+3. Test that you have Postgres and Redis running.
+
+4. Continue to the [set up local environment section](#set-up-local-environment).
+
+### Set up local environment
+
+1. Run the following command to set up your local environment:
+
+    ```
+    $ make setup
+    ```
+
+    This command copies sample configuration files, installs required gems and brew packages (if using macOS), and sets up the database. Check out our [Makefile commands](../Makefile) to learn more about what this command does.
+
+1. Now that you have you have everything installed, you can run the following command to start your local server:
+
+    ```
+    $ make run
+    ```
+
+    You should now be able to go to open up your favorite browser, go to `localhost:3000` and see your local development environment running.
+
+### Running tests locally
+
+  Identity Reporting uses the following tools for our testing:
+
+  - [RSpec](https://relishapp.com/rspec/rspec-core/docs/command-line)
+
+  To run our full test suite locally, use the following command:
+
+  ```
+  $ make test
+  ```
+
+  Use the following command to run a subset of our test suite, excluding slower tests:
+
+  ```
+  $ make fast_test
+  ```
+
+  Check out our Makefile commands and learn more about how you can customize this command to run specific tests using rspec: https://github.com/18F/identity-reporting-rails/blob/main/Makefile
+
+  To test a specific spec file with rspec, you may need to add the following configuration to `/config/application.yml` so the tests do not crash:
+  ```
+
+## Linting
+
+Run `make lint` to look for errors; `make lintfix` can repair some linting errors.
+
+
+## Running jobs
+
+We run background jobs / workers with ActiveJob and GoodJob. You shouldn't normally have to start it manually because `make run` runs [the `Procfile`](Procfile), which handles it. The manual command is: `bundle exec good_job start`
```

Comment:
needs to be a relative link I think

```suggestion
We run background jobs / workers with ActiveJob and GoodJob. You shouldn't normally have to start it manually because `make run` runs [the `Procfile`](../Procfile), which handles it. The manual command is: `bundle exec good_job start`
```